tests/integration/cocoon/test_cocoon_holdings.py

```diff
@@ -470,6 +514,7 @@ def test_load_from_data_frame_holdings_success(
             property_columns=property_columns,
             properties_scope=properties_scope,
             sub_holding_keys=sub_holding_keys,
+            holdings_adjustment_only=holdings_adjustment_only
```

Comment:
Could you add this argument to the docstring for the test?

Reply:
Done.